Model/Config.php

```diff
@@ -7,16 +7,31 @@ class Config
     public function __construct()
     {
         //Get settings array
-        $this->settings = $this->getSettings();
+        $this->settings = $this->get_settings();
         //Set property for each setting
         $this->setProperties();
     }
 
+    /**
+     * Update a Mobbex option
+     * 
+     * @param string $name Name of the option. 
+     * @param mixed $value Value of the option.
+     * 
+     */
+    public function update_option($name, $value)
+    {
+        $saved_values        = $this->get_settings();
+        $saved_values[$name] = $value;
```

Comment:
Para no tener que volver a obtener las opciones desde base de datos, y para mantener `$this->settings` actualizado se puede hacer un `$this->settings[$name] = $value`, en vez de llamar de nuevo a `get_settings`, y luego en el `update_option` del return pasar `$this->settings`.

También hay que actualizar la propiedad usando `$this->{$name}`.